### DISCUSSION TITLE
Unable to connect wifi but using yiapp

### DISCUSSION THREAD

**Discussion by elzikas** [roleoroleo/yi-hack-Allwinner-v2#813] [General]
First of all thanks a lot for your work, 
I've tested AllwinnerV2 in some cams with sweet results, but in none of them I could use the "configure_wifi.cfg.ori" to do the first connection to wifi without using the Yi App
Last try was a pair of Yi 1080p Home RFUS (YYS.2016, those that comes in x4pack), frmw 9.0.19.12_202102241808,  using **y21ga_0.3.1**. May I know if im doing something wrong? 

I edit the configure_wifi like this, using simple quotes around ssid and password, confirmed is 2,4G (no weird characters just letters and numbers, used Ubuntu to format the sdcard and edit/move the files)
_wifi_ssid='My AP 2.4GHz'
wifi_psk='mypassword'_

then change name to _configure_wifi.cfg_, put the SD, turn cam on (first ever boot), and after a while the cam goes to flashing blue.
I can confirm hack is done because I get all these folders _backup_, _Factory.done_, _log_, created in the SD, and a "swapfile". 
also, I guess _configure_wifi.cfg_ file is being used on boot because if I put ssid and password wrong, or with no simple quotes ('), the cam goes to "waiting to connect" loop instead of silent flashing blue. Any case it doesn't connect to my AP.

I'm sharing log folder  but couldn't find any related to errors there, just paths and processes
[log_start_reset.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/14090254/log_start_reset.tar.gz)

**Comment by roleoroleo** [comment]
Don't use quote or double quote.
Just the string.

**Reply by elzikas** [reply]
thank you, 
I did it after cleaning SD card and unzipping firmware again; cam goes yellow for a while, then clicks and 'Wellcome to Yi home camera', and keeps looping 'Waiting to connect', yellow light flashing.
Sharing screenshot of file paths right before inserting SD Card in cam (real ssid and password changed...)
![imagen](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/157960459/6fe292db-f6ba-43b5-877c-56a63d5c84bc)

Restarted cam, yellow light still, after 10 seconds it goes Click ,"wellcome to yi home camera' and 'waiting to connect" with yellow flashing loops again.

though about SD card speed, but it's 8gb sandisk ultra C10

**Reply by roleoroleo** [reply]
The log doesn't show problems and you configuration is ok.
Other users noticed this problem.
Maybe a protocol issue between the cam and your AP.
You should try to pair the cam with the original app.

**Reply by Adoumsoukaya** [reply]
can you share the files with us please?

**Reply by mr-bembel** [reply]
Idem for me...
